glog-src/config.h

```diff
@@ -139,7 +139,7 @@
 #define PACKAGE_VERSION "0.3.2"
 
 /* How to access the PC from a struct ucontext */
-#define PC_FROM_UCONTEXT uc_mcontext->__ss.__rip
+/*#define PC_FROM_UCONTEXT uc_mcontext->__ss.__rip*/
```

Comment:
What's going on here? Is this an upstream change to glog?

Reply:
It seems to be a change to glog sometime over the years - I checked the latest glog source code.